src/services/crawl-tx/crawl_tx.service.ts

```diff
@@ -441,8 +446,35 @@ export default class CrawlTxService extends BullableService {
       const resultInsertMsgs = await TransactionMessage.query()
         .insert(listMsgModel)
         .transacting(transactionDB);
+
+
       this.logger.warn('result insert messages:', resultInsertMsgs);
+      const DIDfiltered = resultInsertMsgs
+        .filter((msg: any) => DID_EVENT_TYPES.includes(msg.type))
```

Comment:
Just a note regarding my comment below about using an enum instead of an array: this can be achievable by doing `Object.values(DID_EVENT_TYPES).includes(msg.type)`